architecture/Solution Architecture.md

```diff
@@ -0,0 +1,314 @@
+
+* NCSC beveiligingsrichtlijnen voor webapplicaties [https://www.ncsc.nl/documenten/publicaties/2019/mei/01/ict-beveiligingsrichtlijnen-voor-webapplicaties](https://www.ncsc.nl/documenten/publicaties/2019/mei/01/ict-beveiligingsrichtlijnen-voor-webapplicaties)
+
+* NCSC beveiligingsrichtlijnen voor mobiele apps [https://www.ncsc.nl/documenten/publicaties/2019/mei/01/ict-beveiligingsrichtlijnen-voor-mobiele-apps](https://www.ncsc.nl/documenten/publicaties/2019/mei/01/ict-beveiligingsrichtlijnen-voor-mobiele-apps)
+
+# Key characteristics
+
+## Decentralised approach
+
+Like most EU countries - the Netherlands believes that a decentralised approach (i.e. no central tracking) best meets the privacy and security expectations of society.
+
+This Proof of Concept therefore focuses on this approach; at the expense of a Central approach.
+
+## Bluetooth Low Energy (BLE)
+
+There are several ways in which contact/encounters between people may be registered on a smartphone; from a simple manual diary to GPS to bluetooth and so on. 
+
+The EU Toolbox recommends the use of Low Energy Bluetooth (BLE).  Although this project keeps open other options - our first ‘Proof of Concept’ version will be BLE based. 
+
+It is well documented that BLE measurements are inaccurate. For this reason there are several parallel activities (both in this project, by other countries and in collaboration with the other member states) to validate and verify this in field tests and to establish the level of accuracy that we can achieve and whether that is sufficient.
+
+It is also known that for BLE on mobile apps on Android and iOS to work reliably in the background, there is a need for either a) relatively low level access (which is not routinely granted by the vendors)  or b) it is required to use the Google/Apple Exposure Notification framework.
+
+## Google/Apple Exposure Notification framework (GAEN) 
+
+Given the timeline (and the desire for cross border interoperability within the EU eHealth network) this ‘‘Proof of Concept’ - a GAEN based approach is currently given priority. 
+
+However it should be noted that alternatives are kept open. We have identified several privacy/security- and practicality issues that will have to be addressed by Google and Apple, or suboptimal workarounds need to be implemented. Based on testing and the international conversations we are having around this subject, we will continuously evaluate if the GAEN path is still the right one.
+
+# Baseline Approach
+
+This chapter describes the core flow that we are following, which is partially dictated by the choice of protocol.
+
+## Recording encounters 
+
+As described in the GAEN document set - the users’ mobile phones are able to record ephemeral IDs of other phones they meet. These are generated from a daily seed key (Temporary Exposure Key, TEK). Details about this can be found in the GAEN documentation, but it boils down to the following:
+
+![image alt text](images/image_0.png)
```

Comment:
Please use proper alt text. Would be great to use PlantText instead, see [this repo](https://github.com/openstate/govhub/blob/470d2b02423316f822c5109a5531f05d4e8ad23a/README.md#system-overview) for example, which is rendered from a [plain text file](https://github.com/openstate/govhub/blob/470d2b02423316f822c5109a5531f05d4e8ad23a/docs/system_overview.puml).

Reply:
Thanks! I will need to fix this later, I'm using a generator for the MD file and this is hard to fix manually at the moment. I've created an issue for it here: https://github.com/minvws/nl-covid19-notification-app-coordination/issues/9